config.py

```diff
@@ -34,6 +34,10 @@ class EnvVars:
         hide_time_to_first_response (bool): If true, the time to first response metric is hidden in the output
         ignore_users (List[str]): List of usernames to ignore when calculating metrics
         labels_to_measure (List[str]): List of labels to measure how much time the lable is applied
+        enable_mentor_count (str): If set to TRUE, compute number of mentors
```

Comment:
Needs to be `bool` now with our other changes.

Reply:
True. That's what I get for only integrating your suggestions through the web UI.

Reply:
We've all done it. 😄